app/src/main/res/values-et-rEE/strings.xml

```diff
@@ -21,41 +21,41 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="vibrate_on_keypress">"Vibreeri klahvivajutusel"</string>
     <string name="sound_on_keypress">"Heli klahvivajutusel"</string>
-    <string name="popup_on_keypress">"Klahvivajutusel kuva hüpik"</string>
+    <string name="popup_on_keypress">"Kuva hüpik klahvivajutusel"</string>
     <string name="settings_screen_preferences">"Eelistused"</string>
-    <string name="settings_screen_key_press">Nupu vajutus</string>
+    <string name="settings_screen_key_press">Klahvivajutus</string>
     <string name="settings_screen_appearance">Välimus</string>
     <string name="settings_screen_theme">"Teema"</string>
-    <string name="settings_system_default">"Süsteemi vaikeväärt."</string>
-    <string name="auto_cap">"Automaatne suurtähtede kasutamine"</string>
-    <string name="auto_cap_summary">"Iga lause esimese sõna kirjutamine suure algustähega"</string>
+    <string name="settings_system_default">"Vaikeväärtus"</string>
+    <string name="auto_cap">"Automaatne suur algustäht"</string>
+    <string name="auto_cap_summary">"Iga lause esimese sõna alustamine suure algustähega"</string>
     <string name="select_language">"Keeled"</string>
-    <string name="locale_name_en_GB">"Inglise (UK)"</string>
-    <string name="locale_name_en_US">"Inglise (USA)"</string>
+    <string name="locale_name_en_GB">"inglise (UK)"</string>
+    <string name="locale_name_en_US">"inglise (USA)"</string>
     <string name="locale_name_es_US">"hispaania (USA)"</string>
-    <string name="locale_name_hi_ZZ">"Hindi-inglise"</string>
-    <string name="locale_name_sr_ZZ">"Serbia (ladina)"</string>
-    <string name="subtype_traditional">"traditsiooniline"</string>
+    <string name="locale_name_hi_ZZ">"hindi-inglise"</string>
```

Comment:
If you want to lowercase the first char you will also have to remove `StringUtils.capitalizeFirstCodePoint(displayName, displayLocale)`.